### ISSUE TITLE
is using a TCR hash with KOS safe?

### ISSUE THREAD

**Issue by themighty1** [emp-toolkit/emp-ot#87]
Hi, emp-ot currently uses a MITCCR hash for breaking correlations after the KOS check acc.to this line https://github.com/emp-toolkit/emp-ot/blob/eb0daf2a7a88c44b419f6d1276dc19e66f80776f/emp-ot/cot.h#L71

However, the security proof of the KOS paper uses a random oracle.

Since we are also implemting a KOS OT extension, we're trying to understand has there been any recent work which proves KOS security with some kind of a CR hash? Or is this a liberty that emp-ot is taking without relying on a formal proof?

Thanks.

**Comment by wangxiao1254** [comment]
Yes it is. TCCR has been shown to be secure under the leakage in KOS
